### ISSUE TITLE
No trading permission

### ISSUE THREAD

**Issue by blablubbabcDEV** [Shopkeepers/Shopkeepers#425]
**Migrated from:** https://dev.bukkit.org/projects/shopkeepers/issues/424

**Originally posted by mel_instagibson (Jun 14, 2016):** 

What steps will reproduce the problem?
1. create a shopkeeper via /shopkeeper trade villager
2. set it for hire with
3. try to let a regular player hire it and hew will get the "you do not haver permission to trade with this shopkeeper" messageIt always worked like this in all earlier versions, also id's dont worklatest version .1.76.

**Comment by blablubbabcDEV** [comment]
**Originally commented by blablubbabc (Jun 15, 2016):** 

I cannot reproduce your issue.
Make sure that you don't have created an admin shop via the command. Only player shops can be set for hire.

And also make sure that the player has the permission to create shops of the type which is for hire. Players can only hire shops which they have the permissions for.
Also shop ids work fine for me. I tested this with the remote command, which however only accepts ids of admin shops, not player shops.
You might also want to recreate the config, or at least the messages. Some message(s) were changed in the past, which might cause the displaying of different types of ids in the shopkeeper list.

            Edited Jun 15, 2016
